langserver/workspace_did_change_watched_files.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/terraform-linters/tflint/tflint"
 )
 
-func (h *handler) workspaceDidChangeWatchedFiles(ctx context.Context, conn *jsonrpc2.Conn, req *jsonrpc2.Request) (result interface{}, err error) {
+func (h *handler) workspaceDidChangeWatchedFiles(ctx context.Context, conn *jsonrpc2.Conn) (result interface{}, err error) {
```

Comment:
the unparam linter found that the req parameter is not being used in the function